app/algorithm/worker-algorithm.ts

```diff
@@ -0,0 +1,11 @@
+// Currently not used
```

Comment:
If it not used let's remove it from repo.

Reply:
Okay if you think so :)